app.py

```diff
@@ -1593,6 +1608,14 @@ def sendEmail(project_id, evaluation_name, show_score):
     """
     project = Permission.query.filter_by(project_id=project_id).first()
     path_to_load_project = "{}/{}/{}".format(base_directory, project.owner, project.project)
+
+    #################
```

Comment:
what does this mean? If it is important to comment this code, then make a descriptive comment